File: java/ConvertBinaryToDecimal.java
// convert binary number to decimal

public class ConvertBinaryToDecimal {
  public static int binToDecimal(String binaryStr) {
    return Integer.parseInt(binaryStr, 2);
  }

  public static void main(String[] args) {
    System.out.println(binToDecimal("1101")); // 13
  }
}

File: c#/ConvertBinaryToDecimal.cs
// convert binary to decimal

public class ConvertBinaryToDecimal
{
	public static int binToDec(string binaryStr)
	{
		return System.Convert.ToInt32(binaryStr, 2);
	}

	public static void Main(string[] args)
	{
		System.Console.WriteLine(binToDec("1101")); // 13
	}
}